docs/docs/API-Reference/api-openai-responses.mdx

```diff
@@ -0,0 +1,303 @@
+| metadata | dict | Additional metadata |
+
+## Example streaming request
+
+```curl
+curl -X POST \
+  "$LANGFLOW_SERVER_URL/api/v1/responses" \
+  -H "x-api-key: $LANGFLOW_API_KEY" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "model": "your-flow-id",
+    "input": "Tell me a story about a robot",
+    "stream": true
+  }'
+```
+
+<details>
+<summary>Response</summary>
+
+```json
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123456, "model": "your-flow-id", "delta": {"content": "Once"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123457, "model": "your-flow-id", "delta": {"content": " upon"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123458, "model": "your-flow-id", "delta": {"content": " a"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123459, "model": "your-flow-id", "delta": {"content": " time"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123460, "model": "your-flow-id", "delta": {"content": "..."}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123461, "model": "your-flow-id", "delta": {}, "status": "completed"}
+```
+
+</details>
+### Streaming response body
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id | string | Unique response identifier |
+| object | string | Always "response.chunk" |
+| created | int | Unix timestamp of chunk creation |
+| model | string | The flow ID that was executed |
+| delta | dict | The new content chunk |
+| status | string | Response status: "completed", "in_progress", or "failed" (optional) |
+
+## Pass global variables to your flows in headers
+
+Global variables allow you to pass dynamic values to your flows that can be used by components within the flow.
+This is useful for passing API keys, user IDs, or any other configuration that might change between requests.
+
+The `/responses` endpoint accepts global variables as custom HTTP headers with the format `X-LANGFLOW-GLOBAL-VAR-{VARIABLE_NAME}`.
+Variable names are automatically converted to uppercase.
+
+This example passes three global variables as headers.
+In your flow, these variables will be available as `API_KEY` (from `X-LANGFLOW-GLOBAL-VAR-API_KEY`), `USER_ID` (from `X-LANGFLOW-GLOBAL-VAR-USER_ID`), and `ENVIRONMENT` (from `X-LANGFLOW-GLOBAL-VAR-ENVIRONMENT`).
+
+```bash
+curl -X POST \
+  "$LANGFLOW_SERVER_URL/api/v1/responses" \
+  -H "x-api-key: $LANGFLOW_API_KEY" \
+  -H "Content-Type: application/json" \
+  -H "X-LANGFLOW-GLOBAL-VAR-API_KEY: sk-..." \
+  -H "X-LANGFLOW-GLOBAL-VAR-USER_ID: user123" \
+  -H "X-LANGFLOW-GLOBAL-VAR-ENVIRONMENT: production" \
+  -d '{
+    "model": "your-flow-id",
+    "input": "Hello"
+  }'
+```
+
+## Retrieve tool call results
+
+When your flow includes tools or function calls, you can get detailed information about their execution by including `"tool_call.results"` in the `include` parameter.
+
+#### Basic Tool Call Response (Default)
```

Comment:
```suggestion
```

I don't think this subheading is necessary unless you also add one for the other option.